src/shared/components/MSKTabs/MSKTabs.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import classnames from 'classnames';
+import ReactChild = React.ReactChild;
+import ReactNode = React.ReactNode;
+import ReactChildren = React.ReactChildren;
+
+interface IMSKTabProps {
+    hidden?:Boolean;
+    id:string;
+    linkText:string;
+    activeId?:string;
+}
+
+export class MSKTab extends React.Component<IMSKTabProps,{}> {
+
+    constructor(props: IMSKTabProps){
+        super(props);
+    }
+
+    render(){
+        const hidden = ((this.props.hidden===true) ? 'hidden' : '');
+        return <div className={classnames({ 'msk-tab':true, 'hidden':(this.props.hidden === true)  })}>{this.props.children}</div>
```

Comment:
on line 22 and 23, the predicate `this.props.hidden === true` is equivalent to the predicate `this.props.hidden`